mne/beamformer/_dics.py

```diff
@@ -63,11 +63,11 @@ def make_dics(info, forward, csd, reg=0.05, label=None, pick_ori=None,
                 filters are computer for the orientation that maximizes
                 spectral power.
 
-    rank : int | 'auto' | None
-        The effective rank of the covariance matrix. If 'auto', the rank will
-        be estimated before regularization is applied. If ``None``, the rank
-        will be estimated after regularization is applied.
-        Defaults to 'auto'.
+    rank : None | False | int
+        The effective rank of the covariance matrix.
+        If None, the rank will be estimated before regularization is
+        applied. If False, the rank will be estimated after regularization
+        is applied. Defaults to ``None``.
```

Comment:
Yes, here it should default to `None` and for the LCMV it should default to `False`

Reply:
Just re-read this and realized that this means there should be no deprecation cycle for DICS. I'll revert the deprecation there